analyzer.go

```diff
@@ -49,12 +49,9 @@ func (a *Analyzer) Analyze() (platform.AnalyzedMetadata, error) {
 			appMeta = platform.LayersMetadata{}
 		}
 
-		if a.Platform.API().AtLeast("0.8") {
```

Comment:
Because of https://github.com/buildpacks/lifecycle/pull/800/files#diff-f664818a668bae8688d032e037ff762c9a7537beec1916963461e36dd1c40dc3R40 the analyzer can be a bit more ignorant of platform API logic, and just be an "orchestrator".